sources/web/datalab/polymer/modules/ApiManager.ts

```diff
@@ -216,6 +216,25 @@ class ApiManager {
   }
 
   /**
+   * Copy an item from source to destination. Item name collisions at the destination
+   * are handled by Jupyter.
+   * @param fromDir source path to copy from
+   * @param toPath destination path to copy to
+   */
+  static copyItem(fromPath: string, toPath: string) {
+    toPath = ApiManager.contentApiUrl + '/' + toPath;
+    const xhrOptions: XhrOptions = {
+      method: 'POST',
```

Comment:
I had assumed the toPath argument was the full path to the destination file. If it is actually the containing directory, as it appears to be in your first link, then please change the variable name or param description to make that clear.